ansible/roles/openslide/tasks/main.yml

```diff
@@ -88,14 +90,19 @@
 
 # libtiff
 - name: Download libtiff
-  get_url:
-    url: http://download.osgeo.org/libtiff/tiff-4.0.6.tar.gz
-    dest: "{{ root_dir }}"
+
+  command: >-
+    wget --retry-connrefused --waitretry=1 --read-timeout=20 http://download.osgeo.org/libtiff/tiff-4.0.6.tar.gz
```

Comment:
When installing to docker containers, we don't have wget installed in Ubuntu 16.04 by default (we do have curl).  We either need to apt-get wget in the common task (or this task) or use curl (or go back to using ansible's get_url with a timeout).